visualizer/examples.js

```diff
@@ -127,16 +187,20 @@
       examples = JSON.parse(value);
     } else {
       examples = domUtil.$$('#sampleExamples pre').map(function(elem) {
-        return elem.textContent;
+        return {
+          text: elem.textContent,
+          startRule: null
+        };
       });
     }
+
     examples.forEach(function(ex) {
-      setExample(addExample(), ex);
+      setExample(addExample(), ex.text, ex.startRule);
     });
 
     // Select the first example.
-    var first = domUtil.$('#exampleList li:first-child');
-    var firstId = first ? first.id : -1;
+    var firstIDDOM = domUtil.$('#exampleList li:first-child');
```

Comment:
Nit: Maybe 'firstEl' or 'firstExample' would be a slightly better name here? (Your call.)